transport/src/main/java/io/netty/channel/ManualIoEventLoop.java

```diff
@@ -95,6 +95,7 @@ public long deadlineNanos() {
     public ManualIoEventLoop(Thread owningThread, IoHandlerFactory factory) {
         this.owningThread = Objects.requireNonNull(owningThread, "owningThread");
         this.handler = factory.newHandler(this);
+        state = new AtomicInteger(ST_STARTED);
```

Comment:
In theory we could make this happen once `run` is first started - but will still mess up with a wakeup happening right before the event loop's start (running on an external thread)